numpy/core/src/umath/fast_loop_macros.h

```diff
@@ -46,14 +46,26 @@ abs_ptrdiff(char *a, char *b)
     npy_intp i;\
     for(i = 0; i < n; i++, ip1 += is1, op1 += os1, op2 += os2)
 
-/** (ip1, ip2) -> (op1) */
-#define BINARY_LOOP\
+#define BINARY_LOOP_BASE\
```

Comment:
This name is rather unfortunate, since we have `BASE_BINARY_LOOP` further down in the file. Perhaps this should be `BINARY_DEFS`, since it doesn't contain a loop at all?